lib/ansiblelint/constants.py

```diff
@@ -1,7 +1,17 @@
 """Constants used by AnsibleLint."""
 import os.path
+import sys
+
+# mypy needs this special condition or it may fail to run
+# https://github.com/python/typeshed/issues/3500#issuecomment-560958608
```

Comment:
This must mention py3.6 specifically because the link only shows the solution and not motivation.

Reply:
Maybe you want me to also include instructions about pressing the left mouse button over the link and reading the entire page, maybe the reader will not know that to do?!

Reply:
link != motivation